react/week1/todo-app/src/components/functions-hooks/datePick.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+//import React from "react";
```

Comment:
you forgot to remove this useless comment.

Reply:
I removed it